test_scripts/API/DriverDistractionCapability/001_get_driver_distraction_capability.lua

```diff
@@ -0,0 +1,54 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
This just covers the preloaded PT, you should include these capabilities in `user_modules/hmi_values.lua` to test how it would interact with the HMI.